challenges/arrayBinarySearch/array-binary-search.js

```diff
@@ -0,0 +1,14 @@
+const binarySearch = (array, key) => {
+    let index = 0;
```

Comment:
no need for this variable. it is the same value a i

Reply:
yup. this hit me while I was laying in bed last night. Totally overthought that.